tests/topotests/bgp_6vpe_ebgp_topo1/test_bgp_6vpe_ebgp_topo1.py

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env python
+# SPDX-License-Identifier: ISC
+
+#
+# test_bfd_vrflite_topo1.py
```

Comment:
Wrong comment here, just drop it at all.

Reply:
done